microprofile/5.0/mpjwt/2.0/22.0.0.1-TCKResults-java11.adoc

```diff
@@ -0,0 +1,303 @@
+:page-layout: certification 
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Jwt Auth 2.0.
+
+== Open Liberty 22.0.0.1 - MicroProfile Jwt Auth 2.0 Certification Summary (Java 11)
+
+* Product Name, Version and download URL (if applicable):
++
+https://repo1.maven.org/maven2/io/openliberty/openliberty-runtime/22.0.0.1/openliberty-runtime-22.0.0.1.zip[Open Liberty 22.0.0.1]
+* Specification Name, Version and download URL:
++
+link:https://download.eclipse.org/microprofile/microprofile-jwt-auth-2.0/microprofile-jwt-auth-spec-2.0.html[MicroProfile Jwt Auth 2.0]
```

Comment:
Hmm this doesn't exist but I can't find the right one, only the RC2
https://download.eclipse.org/microprofile/microprofile-jwt-auth-2.0-RC2/microprofile-jwt-auth-spec-2.0-RC2.html

Reply:
```suggestion
link:https://download.eclipse.org/microprofile/microprofile-jwt-auth-2.0/microprofile-jwt-auth-spec-2.0.html[MicroProfile Jwt Auth 2.0]
```

Reply:
So this link is actually correct ... now that it exists